src/main/java/com/lorepo/icplayer/client/module/limitedreset/LimitedResetView.java

```diff
@@ -69,14 +70,25 @@ public void onClick(ClickEvent event) {
 						
 						presenter.reset(module.getResetOnlyWrongAnswers());
 					}
-					
-					ValueChangedEvent valueEvent = new ValueChangedEvent(module.getId(), "", "resetClicked", "");
-					playerServices.getEventBusService().getEventBus().fireEvent(valueEvent);
+
+					sendResetClickedEvent(module.getId(), context);
```

Comment:
wywołujesz metodę której funkcjonalność jest zakomentowana

Reply:
Chodzi o linijkę 80? To są oznaczenia z GWT, dzięki temu można pisać JavaScript w Javie.